Makefile

```diff
@@ -29,6 +29,7 @@ help:
 	@echo '  push-{service}:      - Push the current local image for the service to ECR'
 	@echo '  deploy-{service}:    - Deploy the specific service using the latest image in the ECR, need to specific environment with ENV'
 	@echo '                         For example: ENV=dev make deploy-bastion'
+	@echo '                       - Restart API server deployment'
```

Comment:
missing 'restart-api' ?

Reply:
oh yeah. Will add in the next PR. Otherwise need to do the review again